src/utils/sources-tree/addToTree.js

```diff
@@ -1,5 +1,7 @@
 // @flow
 
+import { parse } from "url";
```

Comment:
does it make sense to have this in `getURL` to have one module for URL parsing?

Reply:
Not at the moment -- I did not want to affect any other logic that might be in "assumed" in the code.